KonyAccelerate/controllers/mobile/frmFeedbackController/frmFeedbackController.js

```diff
@@ -0,0 +1,131 @@
+        feedbacks.push(feedbackComment);
+        var batch = {
+            "records": feedbacks
+        };
+        this.createRecord(this.objectServiceL, this.dataModelObject, batch, this.successInStoreFeedback.bind(this), this.failInStoreFeedback.bind(this));
+
+    },
+    /**
+     * @function createRecord
+     * @scope private
+     * @description this is generic function to make POST operation to kony storage objects.
+     * @param objectService {String} - Name of Kony Object Service.
+     * @param dataModelObject {String} - Name of Kony dataObject.
+     * @param record {Object}- JSONArray which is used as batch to create records.
+     * @param successCallback {Callback} - SuccessCallback to be invoked when the operation is successful.
+     * @param errorCallback {Callback}- FailureCallback to be invoked when the operation fails.
+     */
+    createRecord: function(objectService, dataModelObject, record, successCallback, errorCallback) {
+        try {
+
+            var sdkClient = new kony.sdk.getCurrentInstance();
+            var objectInstance;
+            if (Object.keys(sdkClient).length !== 0) {
+                objectInstance = sdkClient.getObjectService(objectService, {
+                    "access": "online"
+                });
+            }
+            if (objectInstance === null || objectInstance === undefined) {
+                kony.application.dismissLoadingScreen();
+                throw {
+                    "error": "ConnectionError",
+                    "message": "Please connect app to MF"
+                };
+
+            }
+            var dataObject = new kony.sdk.dto.DataObject(dataModelObject);
+
+            dataObject.setRecord(record);
+
+            var options = {
+                "dataObject": dataObject,
+                "headers": {
+                    "Content-Type": "application/json"
+                },
+            };
+            if (kony.net.isNetworkAvailable(constants.NETWORK_TYPE_ANY)) {
+                objectInstance.create(options, successCallback, errorCallback);
+            }
+        } catch (exception) {
+            kony.application.dismissLoadingScreen();
+            konymp.logger.error(JSON.stringify(exception), konymp.logger.EXCEPTION);
+            throw exception;
+        }
+    },
+    /**
+     * @function successInStoreFeedback
+     * @scope private
+     * @description this function is the success callback of the createRecord function
+     * @param response - contains the info of the response object.
+     */
+    successInStoreFeedback: function(response) {
+        alert("success in storing feedback");
+    },
+    /**
+     * @function failInStoreFeedback
+     * @scope private
+     * @description this function is the failure callback for the createRecord function.
+     * @param error - contains the info of the error object.
+     */
+    failInStoreFeedback: function(error) {
+        alert("failure in storing the feedback");
+    }
+
+
+});
```

Comment:
Add a single line space after the completion of the file

Reply:
Added New line to controller.